docs/documentation/backup_plan.md

```diff
@@ -0,0 +1,189 @@
+Plan de sauvegarde
+=======
+---
+
+
+# I - Configuration de la sauvegarde
+## 1.1 Mise en oeuvre
+
+> **IMPORTANT**
+> 
+> Cette manipulation sera réalisée manuellement sur la console AWS, sélectionner le service IAM
+> * Nous devons créer un nouvel utilisateur "de service" IAM 
+> * Générer une clé d'accès pour cet utilisateur IAM
+>
+> Manipulation manuelle depuis la Console AWS, sélectionner le service S3
+> * Créer l'objet S3 bucket
+> * Autoriser le rôle et les accès à dédier à l'utilisateur IAM
+>
+> Cette étape ne doit pas être répétée. Il parait suffisant de le réaliser manuellement et de bien le documenter ainsi il sera aisé de reproduire le déploiement à l'identique.
```

Comment:
I should be able to provide AWS CLI commands without much effort...

Reply:
We can add them to the doc when you have them.